vscode-trace-extension/src/trace-explorer/trace-utils.ts

```diff
@@ -114,29 +107,29 @@ export const fileHandler =
                     if (trace) {
                         traces.push(trace);
                     } else {
-                        traceLogger.addLogMessage('Failed to open trace: ' + traceName, fileHandler.name);
-                        traceLogger.addLogMessage(
-                            'There may be an issue with the server or the trace is invalid.',
-                            fileHandler.name
+                        traceLogger.showError(
+                            'Failed to open trace: ' +
+                                traceName +
+                                '. There may be an issue with the server or the trace is invalid.'
                         );
+                        return;
```

Comment:
If returning here no experiment is created with a subset of traces. The idea was that for a list of traces pathes in tracesArray, each trace is posted to the server. If not successful, it continues with the next trace in the array. If one or more are successfully posted, the experiment is created with these traces. Below there is a warning message created to the user of failed traces. If no trace was successfully posted the no experiment is created. 

I think we should remove the return here

Reply:
My mistake - fixed!